code/game/objects/structures/railings.dm

```diff
@@ -24,9 +24,17 @@
 	climbable = FALSE
 
 /obj/structure/railing/ComponentInitialize()
+	if(.)
+		return ..()
+	. = ..()
```

Comment:
What is this

Reply:
> What is this

fixed this as per the discussion in gen-dev